### DISCUSSION TITLE
Why is your code not suitable for RTL direction?

### DISCUSSION THREAD

**Discussion by gavriel-adi** [muxinc/media-chrome#807] [Proposals]
When my site, for example, goes into reading mode from right to left, all the controls go wrong and nothing works, I would really appreciate it if you made it work for reading from right to left as well and not just from left to right.
I'm attaching a picture of how it looks on my site that is set to dir="rtl":
![צילום מסך 2024-02-01 142507](https://github.com/muxinc/media-chrome/assets/54378151/48f3ee00-a1d2-4a63-bbc0-89f2056a05b2)
Of course, as soon as I convert the site to LTR, then everything appears fine, as in the following image:
![צילום מסך 2024-02-01 143406](https://github.com/muxinc/media-chrome/assets/54378151/984cb4ee-b3a2-4f0d-a56d-b0b5be517714)

**Comment by cjpillsbury** [comment]
Hey @gavriel-adi appreciate the feedback and your use of media chrome. Ensuring smooth rtl layout would be a great feature/add to media chrome for sure. Unfortunately, as an open source project, we don't always have the bandwidth to get to all of the features we'd like all at once. Contributions are always welcome if you'd like to take a stab at implementing this before we get a chance to! Alternatively/as a stopgap, could you simply explicitly add `dir="ltr"` to either media controller itself or, if you want the control bar to layout rtl, apply those to the specific elements that layout poorly? Here's an example of the two approaches, with a `<video controls>` element to demonstrate how it behaves in the various browsers in `rtl`:

https://codesandbox.io/p/sandbox/modest-darkness-xsk64g
